src/seedu/addressbook/data/AddressBook.java

```diff
@@ -1,9 +1,6 @@
 package seedu.addressbook.data;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Imported classes should always be listed explicitly. Refer to point 3: https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.adoc#package-and-import-statements